app/controllers/buyers/users_controller.rb

```diff
@@ -92,6 +92,6 @@ def user_params
   end
 
   def redirect_back_or_show_detail
-    redirect_back(fallback_location: admin_buyers_account_user_path(account_id: user.account_id, id: user.id))
+    redirect_back_or_to(admin_buyers_account_user_path(account_id: user.account_id, id: user.id))
```

Comment:
Last point here: https://github.com/rails/rails/blob/7-0-stable/actionpack/CHANGELOG.md#rails-700alpha1-september-15-2021

And the change itself: https://github.com/rails/rails/pull/40671/files